### PR TITLE
Update documents due to change of dictionary loading approach

### PR DESCRIPTION
**Pull request by oayvazoglusim**
This PR makes the following changes 

Due to change of dictionary loading approach ( #3066 )
- Updates the INSTALL.md file description
- Updates anet.yml.production.template file
- Adds anet-dictionary.yml.production.template file under documents

#### System admin changes
- [x] anet.yml and anet-dictionary.yml needs change
- [ ] db needs migration
- [x] documentation has changed
- [ ] graphql schema has changed

### Checklist
  - [ ] Described the user behavior in PR body
  - [ ] Referenced/updated all related issues
  - [ ] commits follow a `repo#issue: Title` title format and [these 7 rules](https://chris.beams.io/posts/git-commit/)
  - [ ] commits have a [clean history](https://epage.github.io/dev/commits/), otherwise PR may be squash-merged
  - [ ] Added and/or updated unit tests
  - [ ] Added and/or updated e2e tests
  - [ ] Added and/or updated data migrations
  - [x] Updated documentation
  - [ ] Resolved all build errors and warnings
  - [ ] Opened debt issues for anything not resolved here

### CODE REVIEW

File: docs/anet.yml.production.template [hunk 1]
```diff
@@ -151,253 +151,3 @@ logging:
       archivedFileCount: 2
       logFormat: '[%d{yyyy-MM-dd HH:mm:ss.SSS,UTC}] %p %c: %m%n'
 
-dictionary:
```
Comment (VassilIordanov) [inline docs/anet.yml.production.template:154]: you still need a
```
anetDictionaryName: "anet-dictionary.yml"
```
to be present in `anet.yml.production.template`

File: docs/anet-dictionary.yml.production.template [hunk 2]
```diff
@@ -0,0 +1,250 @@
+dictionary:
```
Comment (VassilIordanov) [inline docs/anet-dictionary.yml.production.template:1]: the difference between `anet-dictionary.yml.production.template` and `anet-dictionary.yml.template` was mostly intended to be in the non-dictionary part. I suggest to not add a `docs/anet-dictionary.yml.production.template` as it is not needed - it should eventually be identical to `docs/anet-dictionary.yml.template`

**Review by VassilIordanov — CHANGES_REQUESTED** [review]
out of exact scope of this PR, but I forgot to comment that on the original PR, so I would like to request it here:
can we change the content of `anet-dictionary.yml` from:
```
dictionary:
  SUPPORT_EMAIL_ADDR: support@example.com

  engagementsIncludeTimeAndDuration: true

  dateFormats:
    email:
      date: d MMMM yyyy
      withTime: d MMMM yyyy @ HH:mm
    excel: d MMMM yyyy
    forms:
      input:
....
```
to:
```
SUPPORT_EMAIL_ADDR: support@example.com

engagementsIncludeTimeAndDuration: true

dateFormats:
  email:
    date: d MMMM yyyy
    withTime: d MMMM yyyy @ HH:mm
  excel: d MMMM yyyy
  forms:
    input:
...
```
it will make `anet-dictionary.yml` less nested and the editing of it a nicer experience.

**Comment by oayvazoglusim** [comment]
> out of exact scope of this PR, but I forgot to comment that on the original PR, so I would like to request it here:
> can we change the content of `anet-dictionary.yml` from:
> 
> ```
> dictionary:
>   SUPPORT_EMAIL_ADDR: support@example.com
> 
>   engagementsIncludeTimeAndDuration: true
> 
>   dateFormats:
>     email:
>       date: d MMMM yyyy
>       withTime: d MMMM yyyy @ HH:mm
>     excel: d MMMM yyyy
>     forms:
>       input:
> ....
> ```
> 
> to:
> 
> ```
> SUPPORT_EMAIL_ADDR: support@example.com
> 
> engagementsIncludeTimeAndDuration: true
> 
> dateFormats:
>   email:
>     date: d MMMM yyyy
>     withTime: d MMMM yyyy @ HH:mm
>   excel: d MMMM yyyy
>   forms:
>     input:
> ...
> ```
> 
> it will make `anet-dictionary.yml` less nested and the editing of it a nicer experience.

fixed

File: docs/anet-dictionary.yml.production.template (hunk 2 above)
Comment (oayvazoglusim) [reply docs/anet-dictionary.yml.production.template]: docs/anet-dictionary.yml.production.template is removed

File: docs/anet.yml.production.template (hunk 1 above)
Comment (oayvazoglusim) [reply docs/anet.yml.production.template:154]: it is added

**Review by VassilIordanov — COMMENTED** [review]
Related to the previous PR, but worth changing here:
Shouldn't we have an ignore annotation on
```
  private Map<String, Object> dictionary;
``` 
in `AnetConfiguration.java`.
We do not want the dictionary to be persisted in `anet.yml` anymore. I can still define a `dictionary: ` that will override the external file

**Review by VassilIordanov — APPROVED** [review]

**Pull request merged**